var/spack/repos/builtin/packages/swiftsim/package.py

```diff
@@ -59,19 +59,15 @@ def setup_environment(self, spack_env, run_env):
         tty.warn('This is needed to clone SWIFT repository')
         spack_env.set('GIT_SSL_NO_VERIFY', 1)
 
-    def install(self, spec, prefix):
-        # Generate configure from configure.ac
-        # and Makefile.am
+    def autoreconf(self, spec, prefix):
         libtoolize()
         aclocal()
         autoconf()
         autogen = Executable('./autogen.sh')
         autogen()
 
-        # Configure and install
-        options = ['--prefix=%s' % prefix,
-                   '--enable-mpi' if '+mpi' in spec else '--disable-mpi',
-                   '--enable-optimization']
-        configure(*options)
-        make()
-        make("install")
+    def config_args(self):
```

Comment:
I like configure_args better. I'm just pointing out that in the AutotoolsPackage it's called configure_args but here it's different.